### DISCUSSION TITLE
Updated igb driver

### DISCUSSION THREAD

**Discussion by kernelmustard** [Security-Onion-Solutions/securityonion#6179] [Ideas]
Our sensors, running 2.3.80, have to have the updated igb driver in order to load balance between workers correctly. Security Onion ships with igb version 5.6.0-k, and we have to download and compile 5.7.2. I don't know if updating this will break other users' builds, but it would be much nicer to have it work out of the box.

This is the driver in question
https://www.intel.com/content/www/us/en/download/14098/13663/intel-network-adapter-driver-for-82575-6-82580-i350-and-i210-211-based-gigabit-network-connections-for-linux.html

**Comment by dougburks** [comment]
We don't include specific drivers in Security Onion. We simply use the drivers that are built into the Linux kernel. I don't believe we've had any other reports like this. A few questions:

I assume you are using Ubuntu 18.04? 

Which specific Intel NIC model are you using? 

What exactly changed from 5.6.0 to 5.7.2 for worker load balancing?

**Comment by vanh20** [comment]
Yes. I have a similar predicament [here](https://github.com/Security-Onion-Solutions/securityonion/discussions/6788). It would be nice to see what does work and what doesn't.

@kernelmustard, did you have problems doing the normal install? How was this finally resolved on your end?
